
---

https://www.sciencefriday.com/segments/65-genomes-expand-our-picture-of-human-genetics/?utm_source=wnyc&utm_medium=podcast&utm_campaign=scifri

---

| WORD           | IPA                                                  |      |       |
| -------------- | ---------------------------------------------------- | ---- | ----- |
| Genome         | uk  /ˈdʒiː.nəʊm/ us  /ˈdʒiː.noʊm/                    | 1    |       |
| draft          | uk  /drɑːft/ us  /dræft/                             |      |       |
| race           | uk  /reɪs/ us  /reɪs/                                |      |       |
| fuller         |                                                      |      |       |
| resequencing   |                                                      |      |       |
| reassembling   | uk  /ˌriː.əˈsem.bəl/ us  /ˌriː.əˈsem.bəl/            | 1    |       |
| lessen         | uk  /ˈles.ən/ us  /ˈles.ən/                          |      |       |
| typos          | uk  /ˈtaɪ.pəʊ/ us  /ˈtaɪ.poʊ/                        |      |       |
| firm           | uk  /fɜːm/ us  /fɝːm/                                |      |       |
| puzzle         | uk  /ˈpʌz.əl/ us  /ˈpʌz.əl/                          |      |       |
| Christine      | /ˈkrɪstiːn/                                          |      |       |
| Beck           | uk  /bek/ us  /bek/                                  |      |       |
| genetics       | uk  /dʒəˈnet.ɪks/ us  /dʒəˈnet̬.ɪks/                 |      |       |
| Connecticut    | uk  /kəˈnet.ɪ.kət/ us  /kəˈnet̬.ɪ.kət/               | 1    |       |
| Laboratory     | uk  /ləˈbɒr.ə.tər.i/ us  /ˈlæb.rə.tɔːr.i/            |      |       |
| Farmington     |                                                      |      |       |
| Glennis        |                                                      |      |       |
| Logsdon        |                                                      |      |       |
| assistant      | uk  /əˈsɪs.tənt/ us  /əˈsɪs.tənt/                    |      |       |
| Epigenetics    | uk  /ˌep.ɪ.dʒəˈnet.ɪks/ us  /ˌep.ɪ.dʒəˈnet̬.ɪks/     |      |       |
| Institute      | uk  /ˈɪn.stɪ.tʃuːt/ us  /ˈɪn.stə.tuːt/               |      |       |
| Pennsylvania   | uk  /ˌpen.sɪlˈveɪ.ni.ə/ us  /ˌpen.səlˈveɪ.ni.ə/      | 1    |       |
| Philly         |                                                      |      |       |
| Adam           | uk  /ˈæd.əm/ us  /ˈæd.əm/                            | 1    |       |
| Phillippy      |                                                      |      |       |
| investigator   | uk  /ɪnˈves.tɪ.ɡeɪ.tər/ us  /ɪnˈves.tə.ɡeɪ.t̬ɚ/      |      |       |
| Genomics       | uk  /dʒəˈnəʊm.ɪks/ us  /dʒəˈnoʊm.ɪks/                |      |       |
| Bethesda       |                                                      |      |       |
| Maryland       | uk  /ˈmeə.ri.lænd/ us  /ˈmer.ə.lənd/                 |      |       |
| assemblies     | uk  /əˈsem.bli/ us  /əˈsem.bli/                      |      |       |
| diverse        | uk  /daɪˈvɜːs/ us  /dɪˈvɝːs/                         |      |       |
| chromosomes    | uk  /ˈkrəʊ.mə.səʊm/ us  /ˈkroʊ.mə.soʊm/              |      |       |
| plaguing       | uk  /pleɪɡ/ us  /pleɪɡ/                              |      |       |
| variation      | uk  /ˌveə.riˈeɪ.ʃən/ us  /ˌver.iˈeɪ.ʃən/             |      |       |
| amongst        | uk  /əˈmʌŋst/ us  /əˈmʌŋst/                          |      | among |
| centromeres    |                                                      |      |       |
| essential      | uk  /ɪˈsen.ʃəl/ us  /ɪˈsen.ʃəl/                      |      |       |
| chromosomal    | uk  /ˌkrəʊ.məˈsəʊ.məl/ us  /ˌkroʊ.məˈsoʊ.məl/        |      |       |
| segregated     | uk  /ˈseɡ.rɪ.ɡeɪt/ us  /ˈseɡ.rə.ɡeɪt/                |      |       |
| mitosis        | uk  /maɪˈtəʊ.sɪs/ us  /maɪˈtoʊ.sɪs/                  |      |       |
| meiosis        | uk  /maɪˈəʊ.sɪs/ us  /maɪˈoʊ.sɪs/                    |      |       |
| repetitive     | uk  /rɪˈpet.ə.tɪv/ us  /rɪˈpet̬.ə.t̬ɪv/              | 2    |       |
| entirety       | uk  /ɪnˈtaɪə.rə.ti/ us  /ɪnˈtaɪr.ə.t̬i/              |      |       |
| traverse       | uk  /trəˈvɜːs/ us  /trəˈvɝːs/                        | 1    |       |
| maximize       | uk  /ˈmæk.sɪ.maɪz/ us  /ˈmæk.sə.maɪz/                |      |       |
| novel          | uk  /ˈnɒv.əl/ us  /ˈnɑː.vəl/                         |      |       |
| discovered     | uk  /dɪˈskʌv.ər/ us  /dɪˈskʌv.ɚ/                     |      |       |
| bunch          | uk  /bʌntʃ/ us  /bʌntʃ/                              |      |       |
| investment     | uk  /ɪnˈvest.mənt/ us  /ɪnˈvest.mənt/                |      |       |
| topology       | uk  /tɒpˈɒl.ə.dʒi/ us  /təˈpɑː.lə.dʒi/               | TODO |       |
| pathogenic     | uk  /ˌpæθ.əˈdʒen.ɪk/ us  /ˌpæθ.əˈdʒen.ɪk/            |      |       |
| diversity      | uk  /daɪˈvɜː.sə.ti/ us  /dɪˈvɝː.sə.t̬i/              |      |       |
| similarity     | uk  /ˌsɪm.ɪˈlær.ə.ti/ us  /ˌsɪm.əˈler.ə.t̬i/         |      |       |
| dissimilarity  |                                                      |      |       |
| cataloged      | uk  /ˈkæt.əl.ɒɡ/ us  /ˈkæt̬.əl.ɑːɡ/                  |      |       |
| versus         | uk  /ˈvɜː.səs/ us  /ˈvɝː.səs/                        |      |       |
| decent         | uk  /ˈdiː.sənt/ us  /ˈdiː.sənt/                      |      |       |
| loci           | uk  /ˈləʊ.saɪ/ us  /ˈloʊ.saɪ/                        |      | locus |
| comprised      | uk  /kəmˈpraɪz/ us  /kəmˈpraɪz/                      |      |       |
| uncover        | uk  /ʌnˈkʌv.ər/ us  /ʌnˈkʌv.ɚ/                       |      |       |
| heretofore     | uk  /ˌhɪə.tuːˈfɔːr/ us  /ˌhɪr.tuːˈfɔːr/              |      |       |
| structural     | uk  /ˈstrʌk.tʃər.əl/ us  /ˈstrʌk.tʃɚ.əl/             |      |       |
| consortium     | uk  /kənˈsɔː.ti.əm/ us  /kənˈsɔːr.ʃəm/               |      |       |
| compendium     | uk  /kəmˈpen.di.əm/ us  /kəmˈpen.di.əm/              |      |       |
| machinery      | uk  /məˈʃiː.nər.i/ us  /məˈʃiː.nɚ.i/                 |      |       |
| tripos         | uk  /ˈtraɪ.pɒs/ us  /ˈtraɪ.pɑːs/                     |      |       |
| analogy        | uk  /əˈnæl.ə.dʒi/ us  /əˈnæl.ə.dʒi/                  |      |       |
| jigsaw         | uk  /ˈdʒɪɡ.sɔː/ us  /ˈdʒɪɡ.sɑː/                      |      |       |
| Telomere       | uk  /ˈtiː.lə.mɪər/ us  /ˈte.lə.mɪr/                  |      |       |
| computational  | uk  /kɒm.pjəˈteɪ.ʃən.əl/ us  /kɑːm.pjəˈteɪ.ʃən.əl/   |      |       |
| sheer          | uk  /ʃɪər/ us  /ʃɪr/                                 |      |       |
| quantity       | uk  /ˈkwɒn.tə.ti/ us  /ˈkwɑːn.t̬ə.t̬i/               |      |       |
| billions       | uk  /ˈbɪl.jən/ us  /ˈbɪl.jən/                        |      |       |
| Obviously      | uk  /ˈɒb.vi.əs.li/ us  /ˈɑːb.vi.əs.li/               |      |       |
| ish            | uk  /ɪʃ/ us  /ɪʃ/                                    |      |       |
| exponentially  | uk  /ˌek.spəˈnen.ʃəl.i/ us  /ˌek.spoʊˈnen.ʃəl.i/     |      |       |
| mutable        | uk  /ˈmjuː.tə.bəl/ us  /ˈmjuː.t̬ə.bəl/               |      |       |
| identical      | uk  /aɪˈden.tɪ.kəl/ us  /aɪˈden.t̬ə.kəl/             |      |       |
| despite        | uk  /dɪˈspaɪt/ us  /dɪˈspaɪt/                        |      |       |
| variability    | uk  /ˌveə.ri.əˈbɪl.ə.ti/ us  /ˌver.i.əˈbɪl.ə.t̬i/    |      |       |
| segregate      | uk  /ˈseɡ.rɪ.ɡeɪt/ us  /ˈseɡ.rə.ɡeɪt/                |      |       |
| stuff          | uk  /stʌf/ us  /stʌf/                                |      |       |
| junk           | uk  /dʒʌŋk/ us  /dʒʌŋk/                              |      |       |
| shine          | uk  /ʃaɪn/ us  /ʃaɪn/                                |      |       |
| modalities     | uk  /məʊˈdæl.ə.ti/ us  /moʊˈdæl.ə.t̬i/               |      |       |
| segmental      | uk  /seɡˈmen.təl/ us  /seɡˈmen.t̬əl/                 |      |       |
| duplications   | uk  /ˌdjuː.plɪˈkeɪ.ʃən/ us  /ˌduː.pləˈkeɪ.ʃən/       |      |       |
| infer          | uk  /ɪnˈfɜːr/ us  /ɪnˈfɝː/                           |      |       |
| mutations      | uk  /mjuːˈteɪ.ʃən/ us  /mjuːˈteɪ.ʃən/                |      |       |
| genes          | uk  /dʒiːn/ us  /dʒiːn/                              |      |       |
| embedded       | uk  /ɪmˈbed.ɪd/ us  /ɪmˈbed.ɪd/                      |      |       |
| cisregulatory  |                                                      |      |       |
| downstream     | uk  /ˌdaʊnˈstriːm/ us  /ˌdaʊnˈstriːm/                |      |       |
| quote          | uk  /kwəʊt/ us  /kwoʊt/                              |      |       |
| unquote        | uk  /ʌnˈkwəʊt/ us  /ʌnˈkwoʊt/                        |      |       |
| genetically    | uk  /dʒəˈnet.ɪ.kəl.i/ us  /dʒəˈnet̬.ɪ.kəl.i/         |      |       |
| AlphaFold      |                                                      |      |       |
| atypical       | uk  /ˌeɪˈtɪp.ɪ.kəl/ us  /ˌeɪˈtɪp.ɪ.kəl/              |      |       |
| forth          | uk  /fɔːθ/ us  /fɔːrθ/                               |      |       |
| tipping        | uk  /tɪp/ us  /tɪp/                                  |      |       |
| mutate         | uk  /mjuːˈteɪt/ us  /mjuːˈteɪt/                      |      |       |
| mutation       | uk  /mjuːˈteɪ.ʃən/ us  /mjuːˈteɪ.ʃən/                |      |       |
| deleterious    | uk  /ˌdel.ɪˈtɪə.ri.əs/ us  /ˌdel.ɪˈtɪr.i.əs/         |      |       |
| noncoding      |                                                      |      |       |
| fraction       | uk  /ˈfræk.ʃən/ us  /ˈfræk.ʃən/                      |      |       |
| majority       | uk  /məˈdʒɒr.ə.ti/ us  /məˈdʒɔː.rə.t̬i/              |      |       |
| benign         | uk  /bɪˈnaɪn/ us  /bɪˈnaɪn/                          |      |       |
| genetic        | uk  /dʒəˈnet.ɪk/ us  /dʒəˈnet̬.ɪk/                   |      |       |
| electrical     | uk  /ɪˈlek.trɪ.kəl/ us  /ɪˈlek.trɪ.kəl/              |      |       |
| optical        | uk  /ˈɒp.tɪ.kəl/ us  /ˈɑːp.tɪ.kəl/                   |      |       |
| suspected      | uk  /səˈspek.tɪd/ us  /səˈspek.tɪd/                  |      |       |
| pathogenicity  |                                                      |      |       |
| announced      | uk  /əˈnaʊns/ us  /əˈnaʊns/                          |      |       |
| hailed         | uk  /heɪl/ us  /heɪl/                                |      |       |
| softball       | uk  /ˈsɒft.bɔːl/ us  /ˈsɑːft.bɑːl/                   |      |       |
| inference      | uk  /ˈɪn.fər.əns/ us  /ˈɪn.fɚ.əns/                   |      |       |
| Mendelian      |                                                      |      |       |
| architecture   | uk  /ˈɑː.kɪ.tek.tʃər/ us  /ˈɑːr.kə.tek.tʃɚ/          |      |       |
| penetrant      |                                                      |      |       |
| tie            | uk  /taɪ/ us  /taɪ/                                  |      |       |
| astronomically | uk  /ˌæs.trəˈnɒm.ɪ.kəl.i/ us  /ˌæs.trəˈnɑː.mɪ.kəl.i/ |      |       |
| murkier        | uk  /ˈmɜː.ki/ us  /ˈmɝː.ki/                          |      |       |
| territory      | uk  /ˈter.ɪ.tər.i/ us  /ˈter.ə.tɔːr.i/               |      |       |
| combinatorics  |                                                      |      |       |
| alleles        | uk  /əˈliːl/ us  /əˈliːl/                            |      |       |
| predisposition | uk  /ˌpriː.dɪs.pəˈzɪʃ.ən/ us  /ˌpriː.dɪs.pəˈzɪʃ.ən/  |      |       |
| environmental  | uk  /ɪnˌvaɪ.rəˈmen.təl/ us  /ɪnˌvaɪ.rəˈmen.t̬əl/     |      |       |
| probably       | uk  /ˈprɒb.ə.bli/ us  /ˈprɑː.bə.bli/                 |      |       |
| plumb          | uk  /plʌm/ us  /plʌm/                                |      |       |
| aggregate      | uk  /ˈæɡ.rɪ.ɡət/ us  /ˈæɡ.rə.ɡət/                    |      |       |
| remiss         | uk  /rɪˈmɪs/ us  /rɪˈmɪs/                            |      |       |
| credit         | uk  /ˈkred.ɪt/ us  /ˈkred.ɪt/                        |      |       |
| informative    | uk  /ɪnˈfɔː.mə.tɪv/ us  /ɪnˈfɔːr.mə.t̬ɪv/            |      |       |
| institute      | uk  /ˈɪn.stɪ.tʃuːt/ us  /ˈɪn.stə.tuːt/               |      |       |
| undergone      | uk  /ˌʌn.dəˈɡəʊ/ us  /ˌʌn.dɚˈɡoʊ/                    |      |       |
| absolutely     | uk  /ˌæb.səˈluːt.li/ us  /ˌæb.səˈluːt.li/            |      |       |

---

IRA FLATOW: This is Science Friday. I’m Ira Flatow. Remember the Human <span style="color:rgb(255, 0, 0)">Genome</span> Project? Well, the initial <span style="color:rgb(255, 0, 0)">draft</span> was declared complete back in 2003, but researchers then realized that one genome doesn’t paint a complete picture of the human <span style="color:rgb(255, 0, 0)">race</span>. So fast forward a decade or so, there came the 1000 Genomes Project, an attempt to expand the picture by sampling people from all over the world with different backgrounds and try to get a <span style="color:rgb(255, 0, 0)">fuller</span> look at how we’re the same or how we’re different.

Writing this month in the journal Nature, two teams of researchers take another look at some of those 1,000 genomes, <span style="color:rgb(255, 0, 0)">resequencing</span>, <span style="color:rgb(255, 0, 0)">reassembling</span> with more advanced techniques to <span style="color:rgb(255, 0, 0)">lessen</span> the number of <span style="color:rgb(255, 0, 0)">typos</span>, and really <span style="color:rgb(255, 0, 0)">firm</span> up how the pieces of the genome <span style="color:rgb(255, 0, 0)">puzzle</span> fit together.

Joining me now to talk about how it’s going are my guests, Dr. <span style="color:rgb(255, 0, 0)">Christine</span> <span style="color:rgb(255, 0, 0)">Beck</span>, associate professor of <span style="color:rgb(255, 0, 0)">genetics</span> and genome sciences at the University of <span style="color:rgb(255, 0, 0)">Connecticut</span> Health Center and the Jackson <span style="color:rgb(255, 0, 0)">Laboratory</span>. She’s in <span style="color:rgb(255, 0, 0)">Farmington</span>, <span style="color:rgb(255, 0, 0)">Connecticut</span>; Dr. <span style="color:rgb(255, 0, 0)">Glennis</span> <span style="color:rgb(255, 0, 0)">Logsdon</span>, <span style="color:rgb(255, 0, 0)">assistant</span> professor of genetics and core member of the <span style="color:rgb(255, 0, 0)">Epigenetics</span> <span style="color:rgb(255, 0, 0)">Institute</span> at the University of <span style="color:rgb(255, 0, 0)">Pennsylvania</span> in <span style="color:rgb(255, 0, 0)">Philly</span>; and Dr. <span style="color:rgb(255, 0, 0)">Adam</span> <span style="color:rgb(255, 0, 0)">Phillippy</span>, a senior <span style="color:rgb(255, 0, 0)">investigator</span> in the Center for <span style="color:rgb(255, 0, 0)">Genomics</span> and Data Science Research at the National Human Genome Research Institute at the NIH in <span style="color:rgb(255, 0, 0)">Bethesda</span>, <span style="color:rgb(255, 0, 0)">Maryland</span>. Welcome all of you to Science Friday.

GLENNIS LOGSDON: Thanks.

ADAM PHILLIPPY: Thanks, Ira. Great to be here.

IRA FLATOW: Nice to have you all. Dr. <span style="color:rgb(255, 0, 0)">Logsdon</span>, let me start with you. What’s the thousand-mile view of this paper? What were you all trying to do here?

GLENNIS LOGSDON: Yeah, the goal of the paper was really to generate complete sequence <span style="color:rgb(255, 0, 0)">assemblies</span> of 65 <span style="color:rgb(255, 0, 0)">diverse</span> human around the world. And we were mainly interested in trying to resolve all sequences within all the 46 <span style="color:rgb(255, 0, 0)">chromosomes</span> within these genomes. And that includes the most challenging regions of the genomes that have been kind of <span style="color:rgb(255, 0, 0)">plaguing</span> scientists for decades to try to resolve their regions.

And the point in doing this is to understand the genetic and <span style="color:rgb(255, 0, 0)">epigenetic</span> <span style="color:rgb(255, 0, 0)">variation</span> of these genomes, to understand how we differ in our sequences, our structures, and trying to understand how proteins also differ <span style="color:rgb(255, 0, 0)">amongst</span> us. Some of the most interesting regions that we resolved are the <span style="color:rgb(255, 0, 0)">centromeres</span>, which are these <span style="color:rgb(255, 0, 0)">essential</span> <span style="color:rgb(255, 0, 0)">chromosomal</span> regions found on every single <span style="color:rgb(255, 0, 0)">chromosome</span> in our genome.

And they’re important for ensuring that our chromosomes are equally and accurately <span style="color:rgb(255, 0, 0)">segregated</span> during <span style="color:rgb(255, 0, 0)">mitosis</span> and <span style="color:rgb(255, 0, 0)">meiosis</span>. And so, for the first time, we were able to solve about 1,200 centromeres from among these 65 genomes and understand how they differ between each of us and what that difference might mean in terms of function.

IRA FLATOW: Does that mean the older genome sequences had problems with them?

GLENNIS LOGSDON: Yes, that’s exactly what that means. Sequencing technologies from about a decade ago weren’t really able to resolve these regions of the genome, and that’s because they’re really highly <span style="color:rgb(255, 0, 0)">repetitive</span> and very large. And the sequencing data that was generated back then was smaller than the size of the repeat itself. But now we’re able to actually resolve these regions in their <span style="color:rgb(255, 0, 0)">entirety</span>, <span style="color:rgb(255, 0, 0)">traverse</span> them from one side of the chromosome to the other, and finally get complete maps at high resolution of these regions of the genome.

IRA FLATOW: Interesting. Dr. Beck, tell me more about the 65 <span style="color:rgb(255, 0, 0)">individuals</span> that Dr. Logsdon talked about. Where did these samples come from?

CHRISTINE BECK: Sure. The 65 samples were actually part of the 1000 Genomes Project. So the samples are from around the world. And basically, with data from previous sequencing projects, we had a good handle on how much variation there was between individual samples and a reference genome. So therefore, we chose cell lines from individuals that would <span style="color:rgb(255, 0, 0)">maximize</span> the amount of <span style="color:rgb(255, 0, 0)">novel</span> sequence variation that was <span style="color:rgb(255, 0, 0)">discovered</span> in our work because if we sequenced a <span style="color:rgb(255, 0, 0)">bunch</span> of individuals that were really, really similar, we’d have less return on our <span style="color:rgb(255, 0, 0)">investment</span> for sequencing each individual person.

So we sequenced these 65 people, and from them, we discovered a large amount of DNA variation from person to person. And part of the reason why that’s important is because we don’t really have a good handle on how much DNA variation there is in some of these complex regions of the genome. So without a good understanding of that kind of background <span style="color:rgb(255, 0, 0)">topology</span> of the genome, it’s really, really hard to separate benign differences in the population from <span style="color:rgb(255, 0, 0)">pathogenic</span>.

IRA FLATOW: So you really did find <span style="color:rgb(255, 0, 0)">diversity</span>, <span style="color:rgb(255, 0, 0)">similarity</span>, <span style="color:rgb(255, 0, 0)">dissimilarity</span> in all these different genomes?

CHRISTINE BECK: So we looked between all of these genomes, and between them, we <span style="color:rgb(255, 0, 0)">cataloged</span> 188,000-plus variations between people that were greater than 50 base pairs in length.

IRA FLATOW: Is that surprising, all those?

CHRISTINE BECK: To a certain degree, it’s not. So previous studies looking at an individual genome <span style="color:rgb(255, 0, 0)">versus</span> the reference assembly, we were able to find a <span style="color:rgb(255, 0, 0)">decent</span> number of variants, but in these complex regions, like <span style="color:rgb(255, 0, 0)">Glennis</span> was talking about in other <span style="color:rgb(255, 0, 0)">loci</span> <span style="color:rgb(255, 0, 0)">comprised</span> largely of repeat sequences, we were able to <span style="color:rgb(255, 0, 0)">uncover</span> vast amounts of differences between individuals in the population that had <span style="color:rgb(255, 0, 0)">heretofore</span> been undiscovered because of the quality of sequencing.

So just as a quick side by side example, just four years ago, we published the human genome <span style="color:rgb(255, 0, 0)">structural</span> variant <span style="color:rgb(255, 0, 0)">consortium</span>– at the time published another study where they cataloged variants between people. And with that sequencing technology, there were almost 2,000 fewer variants between every individual and the reference genome than there are in this recent <span style="color:rgb(255, 0, 0)">compendium</span>.

IRA FLATOW: Wow. Dr. <span style="color:rgb(255, 0, 0)">Phillippy</span>, what is it that is letting you do this work now? Is it better <span style="color:rgb(255, 0, 0)">machinery</span> to actually do sequence? Is it better tools to assemble all the data? What is it?

ADAM PHILLIPPY: Yeah, all of the above, including better computers. You mentioned, at the top, the 1000 Genomes Project, which was initiated almost 20 years ago now. That was the original collection and sequencing of these samples. But it wasn’t just <span style="color:rgb(255, 0, 0)">tripos</span> that we had at the time. There was, if you want to continue the book of life <span style="color:rgb(255, 0, 0)">analogy</span>, entire pages missing from each of these individual genomes. And as was noted by <span style="color:rgb(255, 0, 0)">Glennis</span>, a lot of this was due to repeats. And so these repeating pages, repeating sentences, so to speak, in the genome, are just like when you’re doing a <span style="color:rgb(255, 0, 0)">jigsaw</span> puzzle– hard to put back together again when they’re highly <span style="color:rgb(255, 0, 0)">repetitive</span>.

IRA FLATOW: So how complete do you think you’ve gotten? Are we finished?

ADAM PHILLIPPY: Yeah, I was on a number of years ago talking about this <span style="color:rgb(255, 0, 0)">Telomere</span> to Telomere Project, which was the first completion of an entire human genome. And we estimated, at the time, that that filled in about 8% of what was missing after the initial Human Genome Project from the early 2000s. And I would say that number holds about the same for these genomes. And so for all of genomes presented here, each of them has about 8% more sequence than the initial product of the Human Genome Project from 2003.

And the technology, the sequencing methods are able to read a longer stretch of DNA at a time. That helps. The <span style="color:rgb(255, 0, 0)">computational</span> methods have advanced. And we have better and more accurate methods of putting those puzzles back together again. And just handling this <span style="color:rgb(255, 0, 0)">sheer</span> <span style="color:rgb(255, 0, 0)">quantity</span> of data, generating millions and millions of sequencing reads from all of these individual genomes and putting it back together again, is really only possible with the advance of computing that we’ve seen over the past couple of decades as well.

IRA FLATOW: So how close are we to a final number or a final end to all the sequencing?

ADAM PHILLIPPY: Well, how many <span style="color:rgb(255, 0, 0)">billions</span> of people deep would you like to go? <span style="color:rgb(255, 0, 0)">Obviously</span>, we’re just scratching the surface here, but as Christine said, we’re trying to do it in a way that of maximizes our return on investment. And so we can go into a population of people and pick out the ones that look most different from one another, sequence those first. And then, over time, we start to saturate the amount of variation that we return. So now we’re talking about 50-<span style="color:rgb(255, 0, 0)">ish</span> genomes. In the next year or two, we’ll be talking about thousands of genomes. And if this field just continues to increase <span style="color:rgb(255, 0, 0)">exponentially</span> like it has over the past two decades, yeah, the sky’s the limit.

IRA FLATOW: Now you mentioned that this really helped fill in the bits of the genome that repeat over and over. What does that tell us? Why does it repeat over and over? Is there information there?

GLENNIS LOGSDON: Yeah, there’s absolutely information in the repetitive regions of the genome. And not only just information, there’s function I mentioned earlier the <span style="color:rgb(255, 0, 0)">centromeres</span>. They’re some of the most <span style="color:rgb(255, 0, 0)">mutable</span>, highly dynamic regions of our genome. And they’re so mutable that I haven’t yet seen two centromeres that look <span style="color:rgb(255, 0, 0)">identical</span> across humans. And <span style="color:rgb(255, 0, 0)">despite</span> this <span style="color:rgb(255, 0, 0)">variability</span>, we can see quite a bit of variation that, in fact, affects function. So we find that when certain regions of the centromeres are deleted or expanded or duplicated, this could actually affect the way that the chromosomes <span style="color:rgb(255, 0, 0)">segregate</span> during <span style="color:rgb(255, 0, 0)">meiosis</span> and <span style="color:rgb(255, 0, 0)">mitosis</span>.

IRA FLATOW: Is this <span style="color:rgb(255, 0, 0)">repetitive</span> <span style="color:rgb(255, 0, 0)">stuff</span> what we once called <span style="color:rgb(255, 0, 0)">junk</span> DNA?

GLENNIS LOGSDON: It is. It is exactly what you would call junk DNA. But we know for sure that it’s not junk DNA. In fact, it’s very functional, important regions of our genome. It’s important for life. If we didn’t have these regions of the genome, then we wouldn’t be able to live.

CHRISTINE BECK: I think that that’s kind of an important part to touch on because I think repeats of all classes really <span style="color:rgb(255, 0, 0)">shine</span> with these novel techniques and novel sequencing <span style="color:rgb(255, 0, 0)">modalities</span>, as well as the assemblies. So both the centromeric repeats that <span style="color:rgb(255, 0, 0)">Glennis</span> studied, as well as <span style="color:rgb(255, 0, 0)">segmental</span> <span style="color:rgb(255, 0, 0)">duplications</span> and complex kind of different ways of arraying those puzzle pieces, from beginning to end, have begun to come to light with these new sequences.

And from that, you can <span style="color:rgb(255, 0, 0)">infer</span> whether or not the <span style="color:rgb(255, 0, 0)">mutations</span> or the differences between these people have actually affected the coding sequences of <span style="color:rgb(255, 0, 0)">genes</span> <span style="color:rgb(255, 0, 0)">embedded</span> in these repeats or whether or not it might have changed the <span style="color:rgb(255, 0, 0)">cisregulatory</span> landscape. Like, let’s say, the ability to turn a <span style="color:rgb(255, 0, 0)">gene</span> off or turn it up to 11 is also altered between some of these genomes. So getting a good picture of that <span style="color:rgb(255, 0, 0)">repetitive</span> nature of the underlying sequence is really, really key to understanding differences in function <span style="color:rgb(255, 0, 0)">downstream</span>.

IRA FLATOW: Turning a gene up to 11 is something we haven’t spoken about before. How much data do you need to have to tell if something is, <span style="color:rgb(255, 0, 0)">quote</span> <span style="color:rgb(255, 0, 0)">unquote</span>, “normal” <span style="color:rgb(255, 0, 0)">genetically</span>? I just throw that out to any of you.

ADAM PHILLIPPY: I think that’s a great question. And I think it’s really the power of this type of fundamental knowledge generation that we’re doing in these types of projects. Being trained as a computer scientist, I think a lot from that lens.

And in a similar way that something like <span style="color:rgb(255, 0, 0)">AlphaFold</span> succeeded at protein prediction, based on this foundation of the protein data bank that was decades in progress, we’re building now this foundation of what typical human genomes look like. And I think, in the next few years, we’ll see genomic language models, so to speak, trained on that data and be able to predict associations quite accurately between <span style="color:rgb(255, 0, 0)">atypical</span> sequences and their disease associations.

Exactly how many sequences you need and how many people with diseases and without diseases you need in that training set always depends on the type of the disease, how complex those associations are, and so <span style="color:rgb(255, 0, 0)">forth</span>. But I think we’re rapidly approaching a <span style="color:rgb(255, 0, 0)">tipping</span> point in being able to make very accurate predictions off of this genomic data alone.

IRA FLATOW: And what kinds of predictions are we talking about?

ADAM PHILLIPPY: So imagine, as a thought experiment, we just <span style="color:rgb(255, 0, 0)">mutate</span> a random base in your genome. How well do you think we can predict the effect of that <span style="color:rgb(255, 0, 0)">mutation</span>, whether it will be <span style="color:rgb(255, 0, 0)">deleterious</span> or not? We’re not quite that good at it, compared to some other aspects of prediction.

But with these resources, we’re getting much, much better– in particular in the <span style="color:rgb(255, 0, 0)">noncoding</span> regions of the genome that Dr. Beck was just mentioning. A large <span style="color:rgb(255, 0, 0)">fraction</span> of those mutations– you have many millions of them in your genome compared to a typical reference genome, and the vast <span style="color:rgb(255, 0, 0)">majority</span> of them are <span style="color:rgb(255, 0, 0)">benign</span>. But the few that matter are the important ones. And we’re going to get much better in the coming years at making those predictions and being able to spot, basically at birth with DNA sequencing, those predictions, those variants that will likely result in some form of <span style="color:rgb(255, 0, 0)">genetic</span> disease.

IRA FLATOW: Would I be wrong in assuming, Dr. Phillippy, that, as a computer scientist, you’re using a lot of AI here?

ADAM PHILLIPPY: More and more, it’s embedded into a lot of the things we do. The sequencing technologies that we’re using to read off the DNA are using state-of-the-art AI methods to make a prediction from the <span style="color:rgb(255, 0, 0)">electrical</span> current or the <span style="color:rgb(255, 0, 0)">optical</span> image that you’re seeing to the actual As, Cs, Gs, and Ts. So that translation process uses AI. And yes, these kind of DNA models that I was referring to are also coming of age now, and people are actively using them to make predictions of the <span style="color:rgb(255, 0, 0)">suspected</span> <span style="color:rgb(255, 0, 0)">pathogenicity</span> of a variant that you see in one genome compared to another.

IRA FLATOW: Final question. I’ll send it to you, Dr. Beck. I remember when the Human Genome Project was <span style="color:rgb(255, 0, 0)">announced</span>. It was <span style="color:rgb(255, 0, 0)">hailed</span> as a major breakthrough in helping to cure illnesses down the road. How has that been working out? How would you grade the success so far and looking forward?

CHRISTINE BECK: Oh, nice, a <span style="color:rgb(255, 0, 0)">softball</span>.

[LAUGHTER]

So I think that at the end of the day, I think that the sequencing of the human genome has allowed a lot of <span style="color:rgb(255, 0, 0)">inference</span> into <span style="color:rgb(255, 0, 0)">Mendelian</span> diseases. So the <span style="color:rgb(255, 0, 0)">architecture</span> of diseases that are highly <span style="color:rgb(255, 0, 0)">penetrant</span> in the population, where you have a clear variant and effect– so a cause and effect that you can <span style="color:rgb(255, 0, 0)">tie</span> together very clearly– those things have really been helped <span style="color:rgb(255, 0, 0)">astronomically</span> by the development of the human genome reference sequence.

And then stepping into the more <span style="color:rgb(255, 0, 0)">murkier</span> <span style="color:rgb(255, 0, 0)">territory</span> of complex disease genetics, I think that there’s still a lot of work to be done to figure out the underlying genetic architecture of those diseases and understanding the <span style="color:rgb(255, 0, 0)">combinatorics</span> of <span style="color:rgb(255, 0, 0)">alleles</span> and variants that come together to equal the <span style="color:rgb(255, 0, 0)">predisposition</span> to diseases with <span style="color:rgb(255, 0, 0)">environmental</span> factors added to them.

So I think that getting back to what Dr. Phillippy said earlier, I think that an understanding of this is <span style="color:rgb(255, 0, 0)">probably</span> going to be borne out by a much better understanding of variation in genomes, which we’re gaining with studies like ours, mixed with machine learning approaches to <span style="color:rgb(255, 0, 0)">plumb</span> the depths of those data for variants that might, in <span style="color:rgb(255, 0, 0)">aggregate</span> or individually, contribute to these complex diseases. So long story short, I think that there has been a lot of progress. But I also think, in the future, there’s a lot of work and progress to be done.

ADAM PHILLIPPY: I think I would be <span style="color:rgb(255, 0, 0)">remiss</span> to not give <span style="color:rgb(255, 0, 0)">credit</span> to the initial Human Genome Project that we’re building on here that finished up, as you said, about two decades ago now. And I find it really <span style="color:rgb(255, 0, 0)">informative</span> to look back and realize that project took about 10 years and, in today’s dollars, about $5 billion. Each of these individual genomes that we’re doing now at a better quality can be done in basically a few days for around $5,000. And so just do the simple math. That’s a million-fold reduction in the costs to sequence a human genome thanks to these research investments that have been made over the past 25 years by the NIH and by my home <span style="color:rgb(255, 0, 0)">institute</span>, NHGRI.

And it’s just amazing to reflect on the progress that this field has <span style="color:rgb(255, 0, 0)">undergone</span> over the past 20 years with those investments. And so if you look back at the economic impact on that, there was a study in 2013 that estimated the economic impact of the Human Genome Project at $1 trillion, and that was 10 years ago. Imagine what those returns are now. So this Human Genome Project is just a gift that keeps giving, both in terms of economic terms and in terms of quality of life.

IRA FLATOW: Well, I want to thank all of you for taking time to be with us. This has been <span style="color:rgb(255, 0, 0)">informative</span>. I imagine you’re all very, very hopeful about the future.

CHRISTINE BECK: Yeah, <span style="color:rgb(255, 0, 0)">absolutely</span>.

IRA FLATOW: Please come back and tell us more about where this is heading when you get a chance.

CHRISTINE BECK: Will do, thanks.

ADAM PHILLIPPY: Thanks much, Ira.

GLENNIS LOGSDON: Thank you so much.

IRA FLATOW: You’re welcome. Dr. Adam Phillippy at the National Human Genome Research Institute– that is at NIH in Bethesda. Dr. Christine Beck of the University of Connecticut Health Center and the Jackson Laboratory, and Dr. Glennis Logsdon at the University of Pennsylvania. Thank you all for taking time, as I say, to be with us today.

_Copyright © 2025 Science Friday Initiative. All rights reserved. Science Friday transcripts are produced on a tight deadline by 3Play Media. Fidelity to the original aired/published audio or video file might vary, and text might be updated or amended in the future. For the authoritative record of Science Friday’s programming, please visit the original aired/published recording. For terms of use and more information, visit our policies pages at_ [**_http://www.sciencefriday.com/about/policies/_**](http://www.sciencefriday.com/about/policies/)

---

